portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintlabelview.js

```diff
@@ -82,7 +84,7 @@ var OncoprintLabelView = (function () {
 		    var previous_track_id = getLabelAboveMouseSpace(view, track_group, evt.offsetY, view.dragged_label_track_id);
 		    stopDragging(view, previous_track_id);
 		}
-		view.tooltip.hide();
+		view.tooltip.hideIfNotAlreadyGoingTo(150);
```

Comment:
these tooltip methods are so messy.... if you have any suggestions on how to improve that API please let me know